frontend/src/templates/Field/YesNo/YesNoField.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { composeStories } from '@storybook/testing-react'
+import { fireEvent, render, screen, waitFor } from '@testing-library/react'
+
+import * as stories from './YesNoField.stories'
+
+const { ValidationOptional, ValidationRequired } = composeStories(stories)
+
+describe('ValidationRequired', () => {
+  it('renders error when field is not selected before submitting', async () => {
+    // Arrange
+    render(<ValidationRequired />)
+    const submitButton = screen.getByText('Submit')
+
+    // Act
+    fireEvent.click(submitButton)
+    await waitFor(() => submitButton.textContent !== 'Submitting')
+
+    // Assert
+    // Should show error message.
+    const error = screen.getByText('This field is required')
```

Comment:
should we extract all these strings into a constants file since they will be used across many many tests for many many fields?

Reply:
I was thinking about it, but feels like premature optimizationnnnnnnnnnn. But yea makes sense still, will put in `constants/validation.ts`

Reply:
Done in 39c8ad9